src/components/features.tsx

```
@@ -0,0 +1,249 @@
+        }}
+        className={`flex flex-col relative -top-[25%] h-[450px] w-[60%] ml-8 md:ml-20 rounded-md lg:p-10 sm:p-4 p-4 origin-top`}
+      >
+        <h2 className='text-3xl md:text-4xl text-left font-semibold'>{title}</h2>
+        <div className={`flex h-full mt-5 gap-10`}>
+          <div className={`w-[55%] relative top-[6%]`}>
+            <p className='text-base md:text-lg'>{description}</p>
+            <span className='flex items-center gap-2 pt-2'>
+              <a
+                href={'#'}
+                target='_blank'
+                className='underline cursor-pointer'
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)